pvlib/ivtools/fit_sde.py

```diff
@@ -0,0 +1,437 @@
+"""
+The ``fit_sde`` module contains functions to fit the single diode equation.
+
+Function names should follow the pattern "fit_sde_" + fitting method.
+
+"""
+
+import numpy as np
+
+from utility import schumaker_qspline
```

Comment:
needs to be a fully namescaped path to the module, so change to:

    from pvlib.ivtools.utility import schumaker_qspline

or alternately

    from .utility import schumaker_qspline  # one dot means same package, ie pvlib.ivtools

or even

    from ..ivtools.utility import schumaker_qspline  # two dots means one package above, ie pvlib